src/components/NcAppNavigationCaption/NcAppNavigationCaption.vue

```diff
@@ -139,6 +160,15 @@ export default {
 			required: true,
 		},
 
+		/**
+		 * `id` to set on the inner caption
+		 * Can be used for connecting the `NcActionCaption` with `NcActionList` using `aria-labelledby`.
+		 */
+		captionId: {
+			type: String,
+			default: null,
+		},
```

Comment:
`headingId` for consistency with `isHeading` and `headingLevel`

Reply:
I agree